articles/support/index.md

```diff
@@ -35,51 +37,61 @@ Auth0 offers the following support plans:
     </tr>
+    </tr>
+      <th>Premier Success Support</th>
+      <td>For customers with any subscription plan that have added the Premier Success option. 
   </tbody>
 </table>
 
-### No Support
-
-Customers with Auth0's free subscription plan can seek support through the [Auth0 Community](${auth0_community}). Response times may vary and are not guaranteed.
+### No plan
+If you do not have a support plan subscription, you have access to the Auth0 Community. Response times may vary and are not guaranteed.
 
-:::panel Trial Subscriptions
-New Auth0 customers receive Standard Support during the 22-day trial period (customers may check on how many trial days they have left by logging into the [Dashboard](${manage_url})).
+:::panel Trial subscriptions
+New Auth0 customers receive Standard Support during the 22-day trial period. Customers may check on how many trial days they have left by logging into the [Dashboard](${manage_url}).
 
 At the end of the trial period, customers who have not opted to purchase a paid subscription can use the [Auth0 Community](${auth0_community}) for assistance. They will no longer be able to access or open tickets in the [Support Center](https://support.auth0.com/).
 :::
 
-### Standard Support
+### Standard support
 
-Customers with a paid Auth0 subscription plan receive Standard Support, which offers access to the following channels:
+If you have a paid Auth0 subscription plan, you receive Standard Support, which offers access to the following channels:
 
-* Auth0 Community;
-* Auth0 Support Center.
+* Auth0 Community
+* Auth0 Support Center
 
-The Auth0 Support Center offers ticketed support, where support specialists are available to provide assistance.
+The Auth0 Support Center offers ticketed support where support specialists are available to provide assistance.
 
-### Enterprise Support (with or without Preferred Support)
+### Enterprise and Preferred support
 
-Customers with an Enterprise subscription plan receive extended support hours for outages, quicker response times, and Auth0 Sprint onboarding assistance.
+If you have an Enterprise subscription plan, you receive extended support hours for outages, quicker response times, and Auth0 Sprint onboarding assistance.
 
 Please contact the [Auth0 Sales Team](https://auth0.com/?contact=true) if you have specific support requirements or are interested in the Enterprise Support Plan (with or without the Preferred Support option).
 
-## Support Channels
+### Premier Success support
+
+You can add a Premier Success support option to your subscription. With that option, all your inquiries are funneled to a dedicated team of support engineers who understand your use cases, projects, and plans. Your Premier Support Team will be able to quickly investigate and ultimately resolve your issues sooner. Your support plan will also include: 
+
+* Weekly operations reviews with a dedicated Customer Success Manager
+* Issue tracking dashboard
+* 24/7/365 phone support  
```

Comment:
Lets just make this 24/7 as opposed to 24/7/365. I know Kenny and I had spoken on 24/7 but I am not sure he is committed to 24/7/365